args_test.go

```diff
@@ -41,6 +51,31 @@ func expectSuccess(output string, err error, t *testing.T) {
 	}
 }
 
+func expectErrorAs(err error, target error, t *testing.T) {
+	if err == nil {
+		t.Fatalf("Expected error, got nil")
+	}
+
+	targetType := reflect.TypeOf(target)
+	targetPtr := reflect.New(targetType).Interface() // *SomeError
+	if !errors.As(err, targetPtr) {
+		t.Fatalf("Expected error to be %T, got %T", target, err)
+	}
+}
```

Comment:
Why not using simply using this?

```suggestion
func expectErrorAs(err error, target error, t *testing.T) {
	if err == nil {
		t.Fatalf("Expected error, got nil")
	}

	if !errors.As(err, &target) {
		t.Fatalf("Expected error to be %T, got %T", target, err)
	}
}
```